scriptmodules/emulators/dosbox.sh

```diff
@@ -52,14 +52,18 @@ function install_dosbox() {
 }
 
 function configure_dosbox() {
+    local def=1
+    local launcher_name="+Start DOSBox.sh"
     local needs_synth
+    [[ "$md_id" == "dosbox-sdl2" ]] && launcher_name="+Start DOSBox-SDL2.sh" && def=0
```

Comment:
We don't really do multiple variable setting like this elsewhere - would be more readable with an if/fi